src/PostgREST/ApiRequest.hs

```diff
@@ -281,27 +275,26 @@ csvToJson (_, vals) =
           else JSON.String $ toS str
       )
 
--- | Convert {foo} to [{foo}], leave arrays unchanged
--- and truncate everything else to an empty array.
-pluralize :: JSON.Value -> JSON.Array
-pluralize obj@(JSON.Object _) = V.singleton obj
-pluralize (JSON.Array arr)    = arr
-pluralize _                   = V.empty
+consPayloadJSON :: BL.ByteString -> JSON.Value -> Maybe PayloadJSON
```

Comment:
Initially the name `consPayloadJson` confused me because `cons` makes me think of a list operation (like `cons` and `cdr`). I guess it means "construct?" What about another name like `payloadAttributes` or something? Also the first argument could be of type RequestBody